api/v1/flow/flow.proto

```diff
@@ -156,6 +159,16 @@ message Layer7 {
     }
 }
 
+// Trace contains information about a distributed trace.
+message Trace {
+    TraceContext context = 1;
+}
+
+// TraceContext is metadata about trace spans.
+message TraceContext {
+    string trace_id = 1;
```

Comment:
`trace-id` [is defined as a 16-byte array in the spec](https://www.w3.org/TR/trace-context/#trace-id), any reason to keep in its hex-encoded string representation here?

Reply:
Yes, I think a string is more flexible. The current specification for the `TraceParent` is `0` and defines the trace ID as a 16-byte array. However, a future version may change the trace ID format to be something else which, hopefully, could still be represented as a string. I have added a comment to the field to explain that the trace id is a byte array represented as a hex string.